tools/testbench/topology.c

```diff
@@ -50,7 +50,7 @@ static int tb_register_asrc(struct testbench_prm *tp, struct tplg_context *ctx)
 		asrc->source_rate = tp->fs_in;
 
 	/* load asrc component */
-	if (ipc_comp_new(sof->ipc, ipc_to_comp_new(&asrc)) < 0) {
+	if (ipc_comp_new(sof->ipc, ipc_to_comp_new(comp)) < 0) {
```

Comment:
you can simply remove &, it is double pointers if you add &, no need change name to comp

Reply:
Yes, you are right, `asrc` makes sense here better here even if they point to same struct beginning.

Reply:
Fixed!